src/ApprenticeAan/SFA.DAS.ApprenticeAan.Application/Services/QueryStringParameterBuilder.cs

```diff
@@ -8,6 +8,7 @@ public static class QueryStringParameterBuilder
     public static Dictionary<string, string[]> BuildQueryStringParameters(GetCalendarEventsQuery request)
     {
         var parameters = new Dictionary<string, string[]>();
+        if (request.Keyword.Trim() != string.Empty) parameters.Add("keyword", new[] { request.Keyword });
```

Comment:
The input should be trimmed once at the source (web) and then there should be no need to trim it down the line.

Reply:
ok cool.  I'll use a string.IsNullOrWhitespace here